yarn/datadog_checks/yarn/yarn.py

```diff
@@ -350,6 +352,25 @@ def _yarn_capacity_scheduler_metrics(self, metrics_json, addl_tags, queue_blackl
                 tags.extend(addl_tags)
 
                 self._set_yarn_metrics_from_json(tags, queue_json, YARN_QUEUE_METRICS)
+                for sub_queue_json in queue_json['queues']['queue']:
+                    sub_queue_name = sub_queue_json['queueName']
+
+                    if sub_queue_name in queue_blacklist:
+                        self.log.debug('Sublevel Queue "%s" is blacklisted. Ignoring it', sub_queue_name)
+                        continue
+
+                    queues_count += 1
+                    if queues_count > MAX_DETAILED_QUEUES:
+                        self.warning(
+                            "Found more than 100 sub_queues, will only send metrics on first 100 queues and sub_queues."
+                            "Please filter the sub_queues with the check's `queue_blacklist` parameter"
```

Comment:
```suggestion
                            "Found more than 100 sub_queues, only sends metrics on first 100 queues and sub_queues."
                            "Filter the sub_queues with the check's `queue_blacklist` parameter."
```

Reply:
@alai97 Since this is a log line, is it OK if the first line reads `"Found more than 100 sub_queues, only sending metrics on the first 100 queues and sub_queues."`